newlib/libc/sys/vita/net/gethostbyaddr.c

```diff
@@ -0,0 +1,45 @@
+#include <stdlib.h>
+#include <string.h>
+#include <netdb.h>
+#include <errno.h>
+#include <sys/socket.h>
+#include <psp2/net.h>
+
+#define SCE_ERRNO_MASK 0xFF
+
+struct hostent *gethostbyaddr(const void *__addr, socklen_t __len, int __type) {
+    static struct hostent ent;
```

Comment:
Static local variables are not thread-safe.

Reply:
Hm. Although, standard states that `The gethostbyaddr() function need not be reentrant. A function that is not required to be reentrant is not required to be thread-safe.`
so that's probably okay.